datatypes/papas/Cluster.h

```diff
@@ -0,0 +1,76 @@
+//
```

Comment:
Lets discuss when you are here. I discussed with Benedikt how best to arrange the files and this is what he suggested. With this
All the header files will be in a xxxx/papas/  directory
and all the headers will then be (once I have updated everything)
#include "papas/cluster.h" etc